x/wasm-storage/keeper/msg_server_test.go

```diff
@@ -11,9 +11,15 @@ import (
 )
 
 func (s *KeeperTestSuite) TestStoreDataRequestWasm() {
-	wasm, err := os.ReadFile("test_utils/hello-world.wasm")
+	regWasm, err := os.ReadFile("test_utils/hello-world.wasm")
+	s.Require().NoError(err)
+	regWasmZipped, err := ioutils.GzipIt(regWasm)
+	s.Require().NoError(err)
+
+	oversizedWasm, err := os.ReadFile("test_utils/simple-dr.wasm")
```

Comment:
Maybe we should name the file `oversized.wasm`. Right now it feels a bit silly that a 'simple' WASM binary is considered oversized. 😬